Source/RenderStream/Private/RenderStream.h

```diff
@@ -38,6 +40,8 @@ class FRenderStreamModule : public IModuleInterface
 
     void EnableStats() const;
 
+    TArray<ARenderStreamEventHandler*> m_eventHandlers;
```

Comment:
Do we need to keep these around or can we look them up before each broadcast? 

If so, may need to use smart pointers to handle lifetimes.

Reply:
It's easiest to get them here while we have access to World and store them. I'll have a look at whether we could store smart pointers instead.